cosmos/hub/dym_hub/dym_hub.go

```diff
@@ -341,3 +343,56 @@ func (c *DymHub) FullfillDemandOrder(ctx context.Context,
 	}
 	return c.FullNodes[0].ExecTx(ctx, keyName, command...)
 }
+
```

Comment:
Could you help me add a brief comment about the purpose of this function here?

Reply:
it's from another PR already merged.
This function fulfils a demand order coming from EIBC

Reply:
No, I mean QueryRollappState() and FinalizedRollappStateHeight()